src/styles/_catalog.scss

```diff
@@ -20,3 +22,21 @@
     margin-right: 48px;
   }
 }
+
+@media (min-width: 488px) {
```

Comment:
you can do like
```
.catalog {
  ...some styles

  @media (min-width: 488px) {
    grid-template-columns: repeat(2, 200px);
  }
}
```

etc

Reply:
Hello,
Thank you for the comment. I fixed that.